docs/source/json-api/index.rst

```diff
@@ -32,140 +32,123 @@ We welcome feedback about the JSON API on `our issue tracker
+.. code-block:: shell
 
-Once you have retrieved your access token, you can provide it to the HTTP JSON API by storing it in a file. Give the path to it with the ``--access-token-file`` command line option.
+    daml json-api --ledger-host localhost --ledger-port 6865 --http-port 7575 \
+    --query-store-jdbc-config "driver=org.postgresql.Driver,url=jdbc:postgresql://localhost:5432/test?&ssl=true,user=postgres,password=password,createSchema=false"
 
-If the token cannot be read from the provided path or the Ledger API reports an authentication error (for example due to token expiration), the HTTP JSON API will report the error via logging. The token file can be updated with a valid token, and it will be picked up at the next attempt to send a request.
+.. note:: The JSON API provides many other useful configuration flags, run ``daml json-api --help`` to see all of them.
 
-Example session
-***************
+Authentication and Authorization
+================================
 
-.. code-block:: shell
+The Ledger API and JSON API use the terms Authentication and Authorization to describe different aspects of their operation.
+While these terms may sound similar they mean different things.
 
-    $ daml new iou-quickstart-java --template quickstart-java
-    $ cd iou-quickstart-java/
-    $ daml build
-    $ daml sandbox --wall-clock-time --ledgerid MyLedger ./.daml/dist/quickstart-0.0.1.dar
-    $ daml json-api --ledger-host localhost --ledger-port 6865 --http-port 7575
+The JSON API essentially performs two separate tasks:
 
-Choosing a party
-****************
+1. It talks to the Ledger API to get data it needs to operate, for this it needs to may need to provide an *authentication token* to the Ledger API if the Ledger API requires *authentication*.
+2. It handles the requests coming from one or more Parties, for this each party needs to provide an *authorization token* with each request it sends to the JSON API.
 
-Every request requires you to specify a party and some other settings,
-with a JWT token.  Normal HTTP requests pass the token in an
-``Authentication`` header, while WebSocket requests pass the token in a
-subprotocol.
+Authentication
+--------------
 
-In testing environments, you can use https://jwt.io to generate your
+The authentication token is used exclusively for maintaining the internal list of known packages and templates.
+
+.. note:: At no point should an authentication token be provided to an end user, these are for internal use only.
+
+Every authentication token is different and will depend on what your specific ledger or ledger operator requires.
+The JSON API server requires no access to party-specific data, only access to the ledger identity and package services.
+A token issued for the HTTP JSON API server should contain enough claims to contact these two services but no more than that.
```

Comment:
Meaning no claims, those are both public endpoints. You just need a valid token.

Reply:
Should I remove this then?
```
A token issued for the HTTP JSON API server should contain enough claims to contact these two services but no more than that.
Please refer to your ledger operator's documentation to find out how.
```